sdk/spring/spring-cloud-azure-feature-management-web/README.md

```diff
@@ -0,0 +1,3 @@
+# Spring Cloud for Azure feature management web client library for Java
+
+See: [Spring Cloud Azure Feature Management](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/spring-cloud-azure-feature-management)
```

Comment:
How about using relative path (`../spring-cloud-azure-feature-management`)? Same to all other places.